bin/oref0-get-profile.js

```diff
@@ -35,15 +35,34 @@ if (!module.parent) {
     var cwd = process.cwd()
     var pumpsettings_data = require(cwd + '/' + pumpsettings_input);
     var bgtargets_data = require(cwd + '/' + bgtargets_input);
+
+    if (bgtargets_data.units == 'mmol/L') {
+      /* Convert to mg/dL */
+      bgtargets_data.units = 'mg/dL'
+      for (a in bgtargets_data.targets) {
+         var targetarea = bgtargets_data.targets[a];
```

Comment:
This creates a new variable for the translated targets, but never changes or replaces references to bgtargets_data.